model/xgboost/examples/bostonregression/bostonregression.py

```diff
@@ -0,0 +1,44 @@
+from dffml.feature.feature import Feature, Features
+from dffml.noasync import train, accuracy
+from dffml_model_xgboost.xdgregressor import (
+    XDGRegressorModel,
+    XDGRegressorModelConfig,
+)
+from sklearn.datasets import load_boston
+from sklearn.model_selection import train_test_split
+
+boston = load_boston()
+y = boston["target"]
+X = boston["data"]
```

Comment:
The boston housing dataset has been brought up before. We're trying to stay away from it, how about we go with the diabetes dataset instead. We'll need to rename the filename too of course, and it can probably do without having it's own directory (aka just `model/xgboost/examples/diabetesregression.py` is good)

```suggestion
diabetes = load_diabetes()
y = diabetes["target"]
X = diabetes["data"]
```